modules/setup-systemd-resolved/README.md

```diff
@@ -0,0 +1,63 @@
+# systemd-resolved Setup Script
```

Comment:
Oh, cool addition!

What are the trade-offs between dnsmasq and systemd-resolved?

Reply:
It would be nice I think if you can go through https://github.com/hashicorp/consul/issues/4155 to explain the thought process of adding this.

Reply:
Thanks for the pointer! Could you add a short summary to the README of this module, with a link to this thread for more details?